test/features/orNotEqualsLogic.feature

```diff
@@ -0,0 +1,64 @@
+@OrAndNotEqualsLogic
+Feature: Set of Scenarios testing the functionality of the OR, AND, NOT Equals logic
+
+  Background:
+    Given I have logged in
+
+  Scenario: Case View field hide OR condition using case Text Field value
+    Given I am on the case creation form page for case type 'Conditionals' and event 'Create a case OR'
+    When populate field 'TextField4' with value 'orValue4'
+    And submit create a case and navigate to next 'check your answers' page
+    Then the field 'fixed-list' is NOT visible
+
+  Scenario: Case View field show OR condition using case Text Field value
+    Given I am on the case creation form page for case type 'Conditionals' and event 'Create a case OR'
```

Comment:
I would be more inclined to split this into 2 different steps. 

one defining the case exists (see examples in definitionSteps.js) where we can set up the data for jurisdiction/event there etc) rather than parsing in exact case and event 

Second one navigating to create case page which we can reuse an existing step https://github.com/hmcts/ccd-case-management-web/blob/8a4e7d7e6e3e6c454619c2293c95f377271e628b/test/stepDefinitions/create-caseSteps.js#L38

Reply:
done